samples/outbox/store-and-forward/Version_5/Sender/app.config

```diff
@@ -1,17 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Version_5/Version_6 => shouldn't that be Version_2 and Version_3 since it's sql transport specific?